test/plugin/scenarios/nats-client-scenario/support-version.list

```diff
@@ -0,0 +1,21 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+2.15.6
+2.15.4
+2.15.0
+2.14.2
+2.14.0
```

Comment:
Usually, we keep one only per minor version, unless there are some breaking the plugin codes deal with.

Reply:
OK. I will retain 
2.14.0
2.15.0
2.15.6

Reply:
Keep the latest version in minor version.

Reply:
OK